asocial/options/options.html

```diff
@@ -64,20 +39,32 @@ <h1 class="options-title">Расписание</h1>
                     </fieldset>
                     <fieldset class="form-row">
                         <input class="form-checkbox" type="checkbox" name="network" id="vk">
-                        <label class="form-label checkbox-label label-network" for="vk"><i class="fa fa-vk"></i></label>
+                        <label class="form-label checkbox-label label-network" for="vk"><span class="icon icon-vk"></span></label>
                         <input class="form-checkbox" type="checkbox" name="network" id="facebook">
-                        <label class="form-label checkbox-label label-network" for="facebook"><i class="fa fa-facebook"></i></label>
+                        <label class="form-label checkbox-label label-network" for="facebook"><span class="icon icon-facebook"></span></label>
                         <input class="form-checkbox" type="checkbox" name="network" id="twitter">
-                        <label class="form-label checkbox-label label-network" for="twitter"><i class="fa fa-twitter"></i></label>
+                        <label class="form-label checkbox-label label-network" for="twitter"><span class="icon icon-twitter"></span></label>
                         <input class="form-checkbox" type="checkbox" name="network" id="reddit">
-                        <label class="form-label checkbox-label label-network" for="reddit"><i class="fa fa-reddit"></i></label>
+                        <label class="form-label checkbox-label label-network" for="reddit"><span class="icon icon-reddit"></span></label>
+                    </fieldset>
+                    <button class="btn btn-add hidden" type="submit" id="save-button">Добавить</button>
+                    <fieldset class="edit-buttons-row hidden">
+                        <button class="btn" type="submit" id="edit-button">Изменить</button>
+                        <button class="btn btn-red" type="button" id="delete-button">Удалить</button>
                     </fieldset>
-                    <button class="btn" type="submit">Добавить</button>
                 </form>
             </div>
         </div>
     </div>
 
+    <table class="hidden" id="row-template">
```

Comment:
While the parser does process the contents of the <template> element while loading the page, it does so only to ensure that those contents are valid; the element's contents are not rendered, however.